next/frontend/utils/general.ts

```diff
@@ -105,3 +107,20 @@ export const downloadBlob = (blob: Blob, fileName: string) => {
 export function isDefined<T>(value: T | undefined | null): value is T {
   return value !== undefined && value !== null
 }
+export const getTitle = (
+  uiSchema?: UiSchema | object,
+  formData?: GenericObjectType | null,
+  translationFallback?: string,
+): string => {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+  return (
+    _.get(
+      formData,
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+      uiSchema ? uiSchema['ui:options']?.titlePath : undefined || '__INVALID_PATH__',
```

Comment:
Please use [getUiOptions](https://rjsf-team.github.io/react-jsonschema-form/docs/api-reference/utility-functions/#getuioptionst--any-s-extends-strictrjsfschema--rjsfschema-f-extends-formcontexttype--any) instead of accesing the property.